Fitness/Views/Headers/HomeSectionHeaderView.swift

```diff
@@ -1,17 +1,28 @@
 //
-//  HomeCollectionHeaderView.swift
+//  TodaysClassesHeaderView.swift
 //  Fitness
 //
-//  Created by Joseph Fulgieri on 3/18/18.
+//  Created by Joseph Fulgieri on 10/26/18.
 //  Copyright © 2018 Uplift. All rights reserved.
 //
+
+import Crashlytics
 import UIKit
 
+protocol NavigationDelegate {
+    func viewTodaysClasses()
+}
+
 class HomeSectionHeaderView: UICollectionReusableView {
 
     // MARK: - INITIALIZATION
     static let identifier = Identifiers.homeSectionHeaderView
-    var titleLabel: UILabel!
+    static let height = CGFloat(32)
```

Comment:
very small nitpick but I prefer to do `static let height: CGFloat = 32`

Reply:
True that does look nicer